### PR TITLE
PRL-5550

### PR DESCRIPTION
**Pull request by sealDebolina**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the enable_keep_helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/miam/child-protection/template.njk [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {% block page_content %}
 <span class="govuk-caption-xl">{{section}}</span>
-<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
-<div class="govuk-inset-text"> {{needMoreDetails1}}<br/><br/>{{needMoreDetails2}} </div>
-
+<h1 class="govuk-heading-l govuk-!-margin-bottom-5">{{title}}</h1>
+<p class="govuk-body"> {{needMoreDetails1}}</p>
+<h1 class="govuk-heading-m govuk-!-margin-bottom-3">{{subTitle}}</h1>
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/miam/child-protection/template.njk:7]: Not right to have multiple h1 in a page

File: src/main/steps/c100-rebuild/miam/child-protection/content.ts [hunk 2]
```diff
@@ -53,33 +51,27 @@ const languages = {
...
           value: 'localAuthority',
         },
         {
-          name: 'miam_childProtectionEvidence',
           label: l => l.childProtectionPlan,
+          hint: l => l.childProtectionPlanHint,
           value: 'childProtectionPlan',
         },
         {
           divider: true,
```
Comment (vivek-sekhar) [inline src/main/steps/c100-rebuild/miam/child-protection/content.ts:67]: is the "Or" text showing as per design?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/miam/child-protection/content.ts (hunk 2 above)
Comment (sealDebolina) [reply src/main/steps/c100-rebuild/miam/child-protection/content.ts:67]: yes

File: src/main/steps/c100-rebuild/miam/child-protection/template.njk (hunk 1 above)
Comment (sealDebolina) [reply src/main/steps/c100-rebuild/miam/child-protection/template.njk]: addressed

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**